delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/permissions/PermissionsState.scala

```diff
@@ -22,14 +22,23 @@ sealed trait PermissionsState extends Product with Serializable {
   /**
     * @return the current deprecation status (always false for permissions)
     */
-  def deprecated: Boolean = false
+  final def deprecated: Boolean = false
+
+  /**
+    * @return the schema reference that permissions conforms to
+    */
+  final def schema: ResourceRef = Latest(schemas.permissions)
```

Comment:
This way of encoding the value will not work for automatic derivation of typeclasses like Encoder. Same applies for Acls. Maybe that's ok, but it will generate more code.

Reply:
We will see when we get to the Encoding and Decoding, right?